code/__HELPERS/lists.dm

```diff
@@ -640,3 +640,4 @@ proc/dd_sortedObjectList(list/incoming)
 
 // LAZYING PT 2: THE LAZENING
 #define LAZYREINITLIST(L) LAZYCLEARLIST(L); LAZYINITLIST(L);
+#define LAZYQDELLIST(L) if(L) { for(var/o in L) qdel(o); L.Cut() }

```

Comment:
but uh, Crazy.

It's the same, exact define:

https://github.com/ParadiseSS13/Paradise/blob/8d4cc7b6452d943873c5553a9409088fd383618e/code/__HELPERS/qdel.dm#L3